ovs/extensions/packages/packagefactory.py

```diff
@@ -29,36 +30,86 @@ class PackageFactory(_PackageFactory):
     """
     _logger = Logger('extensions-packages')
 
-    universal_packages = ['arakoon', 'openvstorage', 'openvstorage-backend', 'openvstorage-sdm']
-    ose_only_packages = ['alba', 'volumedriver-no-dedup-base', 'volumedriver-no-dedup-server']
-    ee_only_packages = ['alba-ee', 'volumedriver-ee-base', 'volumedriver-ee-server']
+    def __init__(self):
+        """
+        Initialization method
+        """
+        super(PackageFactory, self).__init__()
 
-    universal_binaries = ['arakoon']
-    ose_only_binaries = ['alba', 'volumedriver-no-dedup-server']
-    ee_only_binaries = ['alba-ee', 'volumedriver-ee-server']
+    @classmethod
+    def get_components(cls):
+        """
+        Retrieve the components which relate to this repository
+        :return: A set of components
+        :rtype: set
+        """
+        return {_PackageFactory.COMP_FWK, _PackageFactory.COMP_SD}
```

Comment:
Use cls.COMP_... here
It will point towards _PackageFactory due to the inheritance